api/controllers/apps_test.go

```diff
@@ -55,8 +60,11 @@ func TestAppShowWithAppNotFound(t *testing.T) {
 
 func TestAppCreate(t *testing.T) {
 	aws := test.StubAws(
-		test.CreateAppStackCycle("application"),
-		test.DescribeAppStackCycle("application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeStackNotFound("convox-test-application"),
+		test.CreateAppStackCycle("convox-test-application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeAppStackCycle("convox-test-application"),
```

Comment:
I'll play around here after doing the reversal. There is still going to be a `DescribeStackNotFound` if `AppCreate` continues using `GetApp`, since that wants to check for both styles... buuut since `AppCreate` is special, maybe it should simply call `DescribeStacks("application")` directly and avoid the just as expected cache miss on `convox-test-application` (you were asked to create it after all!). I'll fix up.